docs/usage/configuration-options.md

```diff
@@ -1652,6 +1652,31 @@ For example to apply a special label for Major updates:
 }
 ```
 
+### replacementName
+
+Use this field to define the name of a replacement package.
+Must be used with `replacementVersion` (see example below).
+You can suggest a new community package rule by editing [the `replacements.ts` file on the Renovate repository](https://github.com/renovatebot/renovate/blob/main/lib/config/presets/internal/replacements.ts) and opening a pull request.
+
+### replacementVersion
+
+Use this field to define the name of a replacement package.
+Must be used with `replacementVersion`.
+For example to replace the npm package `jade` with version `2.0.0` of the package `pug`:
```

Comment:
```suggestion
We recommend you also use the `matchDatasources` config option to specify the datasource, this prevents name collisions from other package managers.
For example to replace the npm package `jade` with version `2.0.0` of the package `pug`:
```

Say you have Python and npm package files in your repo, and you have a package called `pug` that exists on both package manager, I think you'd run into conflicts then?

Do we think the chance of collisions is small enough that we don't need to warn about it?

Reply:
I think that the current docs are adequate